CHANGES.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+### ? - ?
+
+##### Breaking Changes :mega:
+
+- `RasterOverlay::createTileProvider` now receives a reference to `CreateRasterOverlayTileProviderParameters` instead of a large number of individual parameters.
+- The constructor parameters for `RasterOverlayTileProvider` and `QuadtreeRasterOverlayTileProvider` have changed.
+- The `getCredit` method has been removed from `RasterOverlayCreditProvider`. Use `getCredits` instead.
+- Removed unused property `RasterOverlayOptions::subTileCacheBytes`.
```

Comment:
This is to address https://github.com/CesiumGS/cesium-native/issues/1274 ? (If so, let's link it so it auto-closes)

Reply:
Oops I pulled this change out of this PR (because I realized I'd prefer to fix that property rather than removing it) and forgot to remove it from the changelog.